HHS/Head-Start-TTADP
# TTAHUB-410: Add Fade In Save Animation and Add Save Alert to Review and Submit

## Description of change

Add Fade In Save Animation and Add Save Alert to Review and Submit.


## How to test

1. Create a new AR and click SAVE on any of the pages. The fixed alert dialog that appears in the navigator should now have a "Fade In" effect. 

2. Go to the Review and Submit page and click Save Draft. You should now see a save alert at the bottom of the page (the same as other pages).



## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-410


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
